src/Components/Search/NewSearch/SearchBarInput.tsx

```diff
@@ -177,8 +191,33 @@ const SearchBarInput: FC<SearchBarInputProps> = ({
     })
   }
 
+  const ref = useRef<HTMLInputElement | null>(null)
+
+  // Focus the search input on '/' keypress
+  useEffect(() => {
+    const handleKeyUp = ({ target, key }: KeyboardEvent) => {
+      if (!ref.current || key !== "/") return
+
+      const tag =
+        (target && (target as HTMLElement).tagName.toLowerCase()) || ""
+
+      // Ignore if an input has focus
+      if (["input", "textarea", "select"].includes(tag)) {
+        return
+      }
```

Comment:
**question (non-blocking):** Does a `select` fire keyup events?

Reply:
Yep, you can tab into one then start typing which will select corresponding options.